app/views/thread.js

```diff
@@ -7,7 +7,20 @@
 
     F.DefaultThreadView = F.View.extend({
         template: 'views/default-thread.html',
-        className: 'thread default'
+        className: 'thread default',
+
+        events: {
+            'click .button[data]': 'onVideoClick',
+        },
+
+        initialize: function() {
+            F.View.prototype.initialize.apply(this, arguments);
+            this.on('clicked-video', this.onVideoClick);
```

Comment:
Two different events to the same handler?  Is this left over from an older variant?